src/server.ts

```diff
@@ -167,7 +167,7 @@ export class Server {
    * @returns {Promise} Promise that resolves to the fee stats returned by Horizon.
    */
   public async operationFeeStats(): Promise<any> {
-    const cb = new CallBuilder(this.serverURL);
+    const cb = new CallBuilder(URI(this.serverURL as any));
```

Comment:
@morleyzhi I'd do a refactor on a different PR, to pull this value in a getter, that way we don't have to remember to use URI each time we want to pass this around

Reply:
Sounds good. Another option is to have CallBuilder handle converting strings to URI objects.